ios/TextileNode.m

```diff
@@ -12,15 +12,14 @@
 
 #define SYSTEM_VERSION_LESS_THAN(v) ([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] == NSOrderedAscending)
 
-
-@interface Callback : NSObject<MobileCallback>
+@interface TEXCallback : NSObject<MobileCallback>
```

Comment:
Renamed these to not conflict with the class names coming from `ios-textile` for now.